src/Symfony/Component/HttpFoundation/Session/Attribute/NamespacedAttributeBag.php

```diff
@@ -43,7 +43,7 @@ public function __construct($storageKey = '_sf2_attributes', $namespaceCharacter
      */
     public function has($name)
     {
-        $attributes = $this->resolveAttributePath($name);
+        $attributes = &$this->resolveAttributePath($name);
```

Comment:
It can improve perf in some situations, but since this is no perf critical code path, I'd keep the code simple...